README.adoc

```diff
@@ -81,21 +81,3 @@ https://testnet-faucet.mempool.co/
 . Start the taker: +
 . The wallets will sync at the beginning which takes a while depending on your internet connection and computer.
 The swap should happen completely automated (note: this may take a while given testnets are not reliable).
-
-== Appendix
```

Comment:
I actually think that this Appendix would motivate people to try. (Because they can use our nodes...)
What is your motivation for removing this? There is no guarantee that the nodes are running and maintained, but I feel we can still tell people that they exist. If there is a problem people can always come to us and ask (which would be a good thing anyway...).

Reply:
I didn't want to have our nodes publicly available.